### ISSUE TITLE
[Paladin] Seal of Command

### ISSUE THREAD

**Issue by krambulli** [ccshiro/cc-buglist#1805]
The damage of Seal of Command does not currently have a delay of 0.5 seconds. After proccing, the damage of Seal of Command should come after 0.5 seconds, in addition to all procs and other effects from that Seal of Command. 
This would enable SoC to chain-proc, if the damage of SoC procs an extra-attack (e.g. Windfury Totem), which then would be able to proc another SoC, as the ICD of SoC is shorter than 0.5 seconds. (SoC proc -> 0.5 sec later: damage of SoC and extra-attack -> SoC proc)

**Comment by Jumpka** [comment]
Yes, right now SoC can't even proc WF. I'm not sure if it's not a duplicate of https://github.com/ccshiro/cc-buglist/issues/1629
Related to the chain proccing, I'm pretty sure ICD of SoC was confirmed to be 1 sec, making it impossible to chain proc it.
